### DISCUSSION TITLE
My crazyflie 2.1 is failing the self test

### DISCUSSION THREAD

**Discussion by XxA7medxX** [bitcraze/discussions#422] [Q&A]
Hello,
My crazyflie 2.1, which I just bought, is not working as it is failing the self-test by triggering a burst of 5 red LEDs on the m1 led. I would love it If someone told me how to solve this.

**Comment by knmcguire** [comment]
Hi!

Do you mean the self-test as explained on the getting started tutorial? https://www.bitcraze.io/documentation/tutorials/getting-started-with-crazyflie-2-x/#test

Or are there any decks attached to the Crazyflie while you start it up?

**Reply by XxA7medxX** [reply]
Yes, the test that is included in the link you did provide

**Reply by knmcguire** [reply]
Oke that is bad! Could you please send us an email at the email listed here? https://www.bitcraze.io/about/contact/ with the order information.
